database/migrations/2022_11_07_171403_add_config_descriptions.php

```diff
@@ -0,0 +1,116 @@
+		DB::table('configs')->where('key', 'raw_formats')->update(['description' => 'Allowed extra format, will not be processed']);
+		DB::table('configs')->where('key', 'map_display')->update(['description' => 'Display the Map given GPS coordinates']);
+		DB::table('configs')->where('key', 'zip64')->update(['description' => 'Use Zip 64bits instead of 32 bits']);
+		DB::table('configs')->where('key', 'force_32bit_ids')->update(['description' => 'Force 32 bits id in the database']);
+		DB::table('configs')->where('key', 'map_display_public')->update(['description' => 'Allow anonymous users to access the map']);
+		DB::table('configs')->where('key', 'map_provider')->update(['description' => 'Defines the map provider']);
+		DB::table('configs')->where('key', 'map_include_subalbums')->update(['description' => 'Includes pictures of the sub albums on the map']);
+		DB::table('configs')->where('key', 'update_check_every_days')->update(['description' => 'Recurency of Lychee update checks']);
+		DB::table('configs')->where('key', 'has_exiftool')->update(['description' => 'Defines whether exiftool processing is available']);
+		DB::table('configs')->where('key', 'share_button_visible')->update(['description' => 'Share button visibility in the header']);
+		DB::table('configs')->where('key', 'has_ffmpeg')->update(['description' => 'Defines whether ffmpeg processing is available']);
+		DB::table('configs')->where('key', 'import_via_symlink')->update(['description' => 'Use symbolic links instead of copying the original on import from server']);
+		DB::table('configs')->where('key', 'apply_composer_update')->update(['description' => 'Apply composer update on lychee update via web interface']);
+		DB::table('configs')->where('key', 'location_decoding')->update(['description' => 'Use GPS location decoding']);
+		DB::table('configs')->where('key', 'location_decoding_timeout')->update(['description' => 'Timeout for the GPS decoding queries']);
+		DB::table('configs')->where('key', 'location_show')->update(['description' => 'Show location extracted from GPS coordinates']);
+		DB::table('configs')->where('key', 'location_show_public')->update(['description' => 'Anonymous users can acess the extracted location from GPS coordinates']);
+		DB::table('configs')->where('key', 'rss_enable')->update(['description' => 'Enable RSS feed']);
+		DB::table('configs')->where('key', 'rss_recent_days')->update(['description' => 'Display the last X days in the RSS feed']);
+		DB::table('configs')->where('key', 'rss_max_items')->update(['description' => 'Max number of itmes in the RSS feed']);
+		DB::table('configs')->where('key', 'prefer_available_xmp_metadata')->update(['description' => 'Use sidecard if provided instead of exif metadata']);
+		DB::table('configs')->where('key', 'editor_enabled')->update(['description' => 'Enable manual rotation of images']);
+		DB::table('configs')->where('key', 'lossless_optimization')->update(['description' => 'Apply compression on images']);
```

Comment:
Honestly, I don't really know what this option does, but surely the compression is used even without it?

Should it just say something like `Apply additional compression on images`?

Reply:
Fixed [3cbe1e8](https://github.com/LycheeOrg/Lychee/pull/1584/commits/3cbe1e86e3691ea69aaff3dfc1eb95c46e27e52d).